### PR TITLE
Generate light documentation for the sdk

### PR DESCRIPTION
**Pull request by chrhicks**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+| [quotes](quotes.md) | Represents a collection of deliveries and available options for fulfillment of that delivery |
+| [services](services.md) | Specific service rendered by the carrier (fedex ground saturday, ups overnight weekend, etc) |
+| [shipments](shipments.md) | Represents a subset of items from an order grouped into a single physical shipment. Flow provides a generated tracking number to track this shipment across multiple possible hops and carriers. |
+| [shipmentLabels](shipmentLabels.md) | Top-level information on shipment tracking shipment_label and number |
+| [tiers](tiers.md) | Service shipping tier available in this tier gorup. e.g. Standard tier, Express tier, Economy tier |
+| [trackings](trackings.md) | Top level tracking information which contains labels. In cases where shipments are re-labeled, you will see multiple labels for each tracking allowing simple access to see where a shipment is - with which carrier and with the local tracking number |
+| [countries](countries.md) |  |
+| [currencies](currencies.md) | ISO 4217 3-character currency code. See http://www.xe.com/iso4217.php |
```
Comment (mbryzek) [inline docs/README.md:41]: use https://api.flow.io/reference/countries?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+| [languages](languages.md) | ISO 639 2-character language code. See https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes |
```
Comment (mbryzek) [inline docs/README.md:42]: use https://api.flow.io/reference/languages?

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+| [timezones](timezones.md) | Time zone data is provided by the public IANA time zone database. See http://www.iana.org/time-zones |
```
Comment (mbryzek) [inline docs/README.md:44]: use https://api.flow.io/reference/timezones?

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+| [documents](documents.md) |  |
```
Comment (mbryzek) [inline docs/README.md:45]: Add regions now that we have them?

**Comment by mbryzek** [comment]
couple minor suggetions but otherwise lgtm

**Comment by chrhicks** [comment]
@mbryzek This documentation is all from the service.json. We should update the individual apis 😄

**Comment by mbryzek** [comment]
thanks - can you regenerate I think my update was merged this week

**Comment by chrhicks** [comment]
Regenerated... see the [diff](https://github.com/flowcommerce/node-sdk/pull/4/commits/532f935cf529034f49acac9d5a0bb6eca30009aa)

File: README.md [hunk 5]
```diff
@@ -2,10 +2,121 @@
...
+#### Making an authenticated request
+
+Most requests to `api.flow.io` require _Basic Authentication_. Here is how to
+make and authenticated request with JavaScript.
+
+```JavaScript
+import client from '@flowio/node-sdk';
+
+const api = client();
+const token = process.env.FLOW_IO_API_TOKEN;
```
Comment (mbryzek) [inline README.md]: In ruby sdk, It hought it was simpler to use the env var FLOW_TOKEN - would suggest we keep consistent. Do you have strong feelings on naming? If so I'll change ruby sdk. If not can we use  FLOW_TOKEN?

File: README.md [hunk 6]
```diff
@@ -2,10 +2,121 @@
...
+Whatever the parameters for a method, it always ends with an options parameter.
+`fetch` is used under the hood, so anything you add to the `options` object will
+be forwarded to the underlying [node-fetch](https://github.com/bitinn/node-fetch)
+implementation. The following are the most common options:
+
+| option  | Type   | Description |
+| :------ | :----- | :---------- |
+| params  | Object | An object of `paramName`:`paramValue` used as query parameters in the request.
+| body    | string | A JSON string representation of the body, e.g, `JSON.stringify(model_form)`.
+| headers | Object | Used for Basic Authentication and any custom headers that may be required as documented for each method of the API.
```
Comment (mbryzek) [inline README.md]: may say this is rare as I'm not sure we use request headers except for auth

File: README.md [hunk 7]
```diff
@@ -2,10 +2,121 @@
...
+      throw new Error('Was not authorized to get organizations!');
+    default:
+      throw new Error(`Response code[${response.status}] not handled!`);
+  }
+});
+```
+
+Each method of the SDK returns a `Promise` of the response. The response will
+contain two properties: `status` and `result`. `response.status` is the HTTP
+status code of the response. `response.result` is the data returned by the api.
```
Comment (mbryzek) [inline README.md]: "is the data" => "contains the data"?

File: README.md [hunk 8]
```diff
@@ -2,10 +2,121 @@
...
+While switch statements are usually frowned upon, at Flow we feel it is critical
```
Comment (mbryzek) [inline README.md]: this is interesting - curious why people frown upon switch statements in node? May be better to remove the negative sentiment from our docs and highlight that we use the http status code to indicate succes and any error conditions. Might be good to introduce the std response codes we use  - namely 401, 404, 422 here

**Comment by mbryzek** [comment]
couple minor suggestions but lgtm

**Pull request merged**